SPECS/libsmi/libsmi.spec

```diff
@@ -80,9 +79,10 @@ make check ||:
 
 
 %files
-%doc ANNOUNCE ChangeLog COPYING README THANKS TODO
+%license COPYING
+%doc ANNOUNCE ChangeLog README THANKS TODO
 %doc doc/draft-irtf-nmrg-sming-02.txt smi.conf-example
-%doc IETF-MIB-LICENSE.txt
+%license IETF-MIB-LICENSE.txt
```

Comment:
Let's just append this to the first `%licence` macro above. That's what we tend to do in most specs.